### PR TITLE
Update from upstream to support rules

### PR DESCRIPTION
**Pull request by charlesverdad**
Pull from upstream to support sentry rules (used for terraforming alerts)

### CODE REVIEW

**Comment by charlesverdad** [comment]
@fa93hws fyi i haven't tested this yet. Would you know of a way to test/import a go branch in terraform?

**Comment by fa93hws** [comment]
> @fa93hws fyi i haven't tested this yet. Would you know of a way to test/import a go branch in terraform?

`export SENTRY_KEY=xxx && export SENTRY_TEST_ORGANIZATION=canva && make test` IIRC

File: go.mod [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
+	google.golang.org/appengine v1.6.5 // indirect
+	google.golang.org/genproto v0.0.0-20191216205247-b31c10ee225f // indirect
+	google.golang.org/grpc v1.26.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999 // indirect
 
-go 1.12.5
+go 1.13
```
Comment (fa93hws) [inline go.mod:38]: we only have go 1.12.x unfortunately.

**Comment by charlesverdad** [comment]
>export SENTRY_KEY=xxx && export SENTRY_TEST_ORGANIZATION=canva && make test IIRC

Oh I meant testing if it works with our `tf -f prod plan` command. I tried modifying `init_providers.go` to use the original repo (non-forked) but it didn't seem to have detected any changes.

**Comment by fa93hws** [comment]
> > export SENTRY_KEY=xxx && export SENTRY_TEST_ORGANIZATION=canva && make test IIRC
> 
> Oh I meant testing if it works with our `tf -f prod plan` command. I tried modifying `init_providers.go` to use the original repo (non-forked) but it didn't seem to have detected any changes.

No, you may have to release to let the bazel make a new build.

**Comment by charlesverdad** [comment]
> No, you may have to release to let the bazel make a new build.

Ah, okay I'll try that. Although I have to merge this first before creating a release, right? I'll ask around to see if there's another way.

**Comment by charlesverdad** [comment]
@fa93hws I noticed you removed the dependency for `github.com/jianyuan/go-sentry`
and this PR introduces it back. Should I remove it ?

**Comment by fa93hws** [comment]
> @fa93hws I noticed you removed the dependency for `github.com/jianyuan/go-sentry`
> and this PR introduces it back. Should I remove it ?

Hi, `go-sentry` has been forked into this repo as `sentryclient`

**Comment by charlesverdad** [comment]
@fa93hws I already tested this and it works! 😄 
Also we now use go `1.13`

File: go.mod (hunk 1 above)
Comment (charlesverdad) [reply go.mod:38]: We now use 1.13 :D

**Review by fa93hws — APPROVED** [review]

**Pull request merged**